kubewarden-dashboard.json

```diff
@@ -1555,8 +1639,8 @@
       {
         "current": {
           "selected": false,
-          "text": "psp-user-group",
-          "value": "psp-user-group"
+          "text": "privileged-pods-policy",
+          "value": "privileged-pods-policy"
```

Comment:
I was wondering this on the previous version of the dashboard. Should we default the dashboard to a specific policy, or provide it "clean"?

Reply:
Yeah, an empty value is better. I just updated the dashboard removing the variable default value.